utils/runtime-fuzzer/src/gear_calls.rs

```diff
@@ -424,32 +421,35 @@ fn config(
         .into_iter(),
     );
 
-    let mut params_config = SyscallsParamsConfig::default();
-    params_config.add_rule(
-        ParamType::Regular(RegularParamType::Alloc),
-        (10..=20).into(),
-    );
-    params_config.add_rule(
-        ParamType::Regular(RegularParamType::Free),
+    let mut params_config = SyscallsParamsConfig::default_regular();
+    params_config.set_rule(RegularParamType::Alloc, (10..=20).into());
+    params_config.set_rule(
+        RegularParamType::Free,
         (initial_pages..=initial_pages + 35).into(),
     );
+    params_config.set_ptr_rule(PtrParamAllowedValues::Value(0..=0));
 
-    let existing_addresses = NonEmpty::collect(
+    let syscall_destination = NonEmpty::collect(
```

Comment:
you sure that should be non empty now?

Reply:
Absolutely. From business logic perspective it's expected to have non empty collection. Otherwise it will make redundant complexity on the `wasm-gen` level